packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsTagsSection.tsx

```diff
@@ -42,7 +42,7 @@ export const FeedSettingsTagsSection = (): ReactElement => {
   const { onUnfollowTags } = useTagAndSource({
     origin: Origin.CustomFeed,
     feedId: feed?.id,
-    shouldFilterLocally: true,
+    shouldFilterLocally: false,
```

Comment:
Yeah, it used to be click "Save" to apply changes, but then we changed it and this was forgotten.